wrong number of episodes burst_to_episode

error can be observed in #73 , @RobinGuillard could you fix that ?
EDIT: got AGAIN the problem #75 

test (edited)
```
def test_burst_to_episode_withparam():
    bursty = [burst(1, 2), burst(3, 4)]
    npt.assert_equal(len(burst_to_episode(bursty,  delim=0, min_burst_joining=1)), 2)
```

result :
```
=================================== FAILURES ===================================
_______________________ test_burst_to_episode_withparam ________________________

    def test_burst_to_episode_withparam():
        bursty = [burst(3.1, 3.9), burst(5, 6),
                  burst(6.5, 7.5), burst(15, 20)]
    
        npt.assert_equal(len(burst_to_episode(bursty,  delim=0, min_burst_joining=1)), 4)
        npt.assert_equal(len(burst_to_episode(bursty,  delim=1, min_burst_joining=3)), 1)
        npt.assert_equal(len(burst_to_episode(bursty, delim=3, min_burst_joining=3)), 2)
        npt.assert_equal(len(burst_to_episode(bursty, delim=1, min_burst_joining=4)), 1)
    
        bursty = [burst(1, 2), burst(3, 4)]
>       npt.assert_equal(len(burst_to_episode(bursty,  delim=0, min_burst_joining=1)), 2)
E       AssertionError: 
E       Items are not equal:
E        ACTUAL: 1
E        DESIRED: 2

test_scoring.py:52: AssertionError
```

--- a/tests/test_scoring.py
+++ b/tests/test_scoring.py
@@ -43,7 +43,7 @@ def test_burst_to_episode_withparam():
     bursty = [burst(3.1, 3.9), burst(5, 6),
               burst(6.5, 7.5), burst(15, 20)]
 
-    npt.assert_equal(len(burst_to_episode(bursty,  delim=1, min_burst_joining=1)), 4)
+    npt.assert_equal(len(burst_to_episode(bursty,  delim=0, min_burst_joining=1)), 4)
     npt.assert_equal(len(burst_to_episode(bursty,  delim=1, min_burst_joining=3)), 1)
     npt.assert_equal(len(burst_to_episode(bursty, delim=3, min_burst_joining=3)), 2)
     npt.assert_equal(len(burst_to_episode(bursty, delim=1, min_burst_joining=4)), 1)

--- a/tests/test_scoring.py
+++ b/tests/test_scoring.py
@@ -43,6 +43,7 @@ def test_burst_to_episode_withparam():
     bursty = [burst(3.1, 3.9), burst(5, 6),
               burst(6.5, 7.5), burst(15, 20)]
 
+    npt.assert_equal(len(burst_to_episode(bursty,  delim=1, min_burst_joining=1)), 4)
     npt.assert_equal(len(burst_to_episode(bursty,  delim=1, min_burst_joining=3)), 1)
     npt.assert_equal(len(burst_to_episode(bursty, delim=3, min_burst_joining=3)), 2)
     npt.assert_equal(len(burst_to_episode(bursty, delim=1, min_burst_joining=4)), 1)

